test/unittests/tar_files.h

```diff
@@ -0,0 +1,268 @@
+/**
```

Comment:
It might make sense to move this file to a sub directory of unittest, like unittest/aux